scripts/cmake/vcpkg_copy_tool_dependencies.cmake

```
@@ -43,5 +58,16 @@ function(vcpkg_copy_tool_dependencies TOOL_DIR)
         endmacro()
         search_for_dependencies("${CURRENT_PACKAGES_DIR}/bin")
         search_for_dependencies("${CURRENT_INSTALLED_DIR}/bin")
+        
+        if (arg_DEPENDENCIES)
+            foreach (SEARCH_ITEM IN_LIST ${arg_DEPENDENCIES})
+                if (EXISTS "${CURRENT_PACKAGES_DIR}/bin/${SEARCH_ITEM}")
+                    debug_message("Copying file ${CURRENT_PACKAGES_DIR}/bin/${SEARCH_ITEM} to ${arg_TOOL_DIR}")
+                    file(COPY "${CURRENT_PACKAGES_DIR}/bin/${SEARCH_ITEM}" DESTINATION "${arg_TOOL_DIR}")
+                else()
+                    message(WARNING "Dynamic dependency ${SEARCH_ITEM} not found in ${CURRENT_PACKAGES_DIR}/bin.")
```

```suggestion
                    message(FATAL_ERROR "Dynamic dependency ${SEARCH_ITEM} not found in ${CURRENT_PACKAGES_DIR}/bin.")
```
This should not be a warning, because the port should have perfect information about whether it calls `LoadLibrary()`. For transitive cases where the port doesn't have perfect information, `applocal.ps1` should be extended to correctly handle those calls.

---

I'm worried that due to some configuration reasons, some dynamic libraries will not be generated in the _bin_ folder and this error will be reported in the future.